core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/TrxRegionEndpoint.java.tmpl

```diff
@@ -4241,8 +4257,12 @@ CoprocessorService, Coprocessor {
                  e.add(kv);
                  try {
                     final WALKey wk = new WALKey(this.regionInfo.getEncodedNameAsBytes(), this.regionInfo.getTable(), EnvironmentEdgeManager.currentTime());
+#ifdef CDH5.7 APACHE1.2
+                    txid = this.tHLog.append(this.m_Region.getTableDesc(), this.regionInfo, wk, e, false);
```

Comment:
The sequenceID gets missing here during logging. Please confirm this does not have side effects during recovery.